controllers/volumehandler/volumehandler.go

```diff
@@ -510,3 +515,8 @@ func (vh *VolumeHandler) pvcFromSnapshot(ctx context.Context, log logr.Logger,
 	logger.V(1).Info("pvc from snap reconciled", "operation", op)
 	return pvc, nil
 }
+
+func (vh *VolumeHandler) IsCopyMethodDirect() bool {
+	return vh.copyMethod == volsyncv1alpha1.CopyMethodDirect ||
```

Comment:
@JohnStrunk Why is there `|| vh.copyMethod == volsyncv1alpha1.CopyMethodNone` here?

Are we trying to check for CopyMethodNone here as well?

Reply:
@kaovilai None was the old name for Direct so the OR is there for backwards compatibility.